src/build/build.ts

```diff
@@ -34,7 +34,7 @@ const buildDirectory = 'build/';
 
 export interface BuildOptions {
   swPrecacheConfig?: string;
-  insertDependencyLinks?: boolean;
+  prefetchDepedencies?: boolean;
```

Comment:
Good point, changed naming to `addPrefetchLinks` to be in line with the cli flag.